src/components/interface/dialog/DialogHeader.jsx

```diff
@@ -2,24 +2,26 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { withStyles } from '@material-ui/core/styles';
 import EditIcon from '@material-ui/icons/Edit';
-import MoreVertIcon from '@material-ui/icons/MoreVert';
 import DialogTitle from '@material-ui/core/DialogTitle';
```

Comment:
Nit, can we group this with the `@material-ui` stuff, move the rest down a line / this up a line.